package.json

```diff
@@ -10,14 +10,19 @@
     "typescript",
     "xrm"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/coveo/coveo-xrm"
+  },
   "homepage": "https://github.com/coveo/coveo-xrm",
   "main": "bin/js/coveo-xrm.min.js",
   "types": "bin/js/src/Index.d.ts",
   "scripts": {
+    "setup": "npm install && npm run build",
```

Comment:
I don't think the `setup` script should run the build. In fact, I would do it the other way around. In my opinion, it would make more sense to have the `build` script do `npm run setup` first. What do you think?